src/docs/tools/doxfilter

```diff
@@ -27,4 +27,4 @@
 
 tooldir=`dirname $0`
 
-python $tooldir/doxfilter.py "$@"
+cat "$@" | python $tooldir/doxfilter.py
```

Comment:
While I see below that you removed a lot of code from `doxfilter.py`, the change above is the only one that is not completely straightforward. I see below that you removed the loop processing `sys.argv[]` and instead just use `stdin`. That change seems unrelated to java.

Reply:
Reverted the change to use `sys.arg[]` as earlier.